internal/gatewayapi/backendtrafficpolicy.go

```diff
@@ -176,28 +176,43 @@ func (t *Translator) ProcessBackendTrafficPolicies(resources *resource.Resources
 				} else {
 					// merge with parent target policy if exists
 					for _, gwNN := range routeParents.UnsortedList() {
+						ancestorRef := getAncestorRefForPolicy(gwNN, nil)
 						// find policy for Gateway
 						gwPolicy := gatewayPolicyMap[gwNN]
 						if gwPolicy == nil {
 							// not found, fall back to the current policy
-							if err := t.translateBackendTrafficPolicyForRoute(policy, route, xdsIR, resources); err != nil {
-								status.SetTranslationErrorForPolicyAncestors(&policy.Status,
-									ancestorRefs,
+							if err := t.translateBackendTrafficPolicyForRoute(policy, route, xdsIR, resources, &gwNN); err != nil {
+								status.SetConditionForPolicyAncestor(&policy.Status,
+									ancestorRef,
 									t.GatewayControllerName,
-									policy.Generation,
+									gwapiv1a2.PolicyConditionAccepted, metav1.ConditionFalse,
+									egv1a1.PolicyReasonInvalid,
 									status.Error2ConditionMsg(err),
+									policy.Generation,
 								)
 							}
+
+							status.SetConditionForPolicyAncestor(&policy.Status,
+								ancestorRef,
+								t.GatewayControllerName,
+								egv1a1.PolicyConditionMerged,
+								metav1.ConditionTrue,
+								egv1a1.PolicyReasonMerged,
+								"Merged with parent policy but not found.",
+								policy.Generation,
```

Comment:
imo lets skip adding a merge for this case, because no merge happened, else will confuse user